### ISSUE TITLE
Staging Review finding: `role="alert"` wrapping warning alert

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98478]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IIR
**Product:** Veteran New Account Flow within Authenticated Experience
**Feature:** Welcome Pathway V1

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/review-information/contact-information](https://staging.va.gov/my-va/welcome-va-setup/review-information/contact-information) when a new user doesn't have a mobile phone number saved, there's a warning alert that reads: "Your mobile phone is missing. Please edit and update the field." This warning alert is wrapped in a `div` which has `role="alert"` as an attribute.

[`role="alert"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role) is equivalent to an `aria-live="assertive"`, which forces the content within that element to be announced immediately by the screen reader, before anything else. It's pretty intrusive for the screen reader experience and is intended for critical, time-sensitive information. When misused, it can be disorienting for screen reader users since the alert content disrupts the normal flow of the page.

In the VA Design System, `role="alert"` is automatically included with error-style alerts (provide immediate, timely information about a negative consequence or a user error) but it's deliberately not included with warning alerts, info alerts, etc. The standard for interrupting the normal flow of the page should be high, and I don't think telling a new user they haven't done something yet meets that standard.

<details><summary>Link, screenshot or steps to recreate</summary>

![Add your contact information page, with "Your mobile phone number is missing" warning alert.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82257-Brian-1.png?raw=true)

Currently coded as:
```
<div class="vads-u-margin-top--1p5" role="alert">
   <va-alert status="warning" ...>...</va-alert>
</div>
```

</details>

## Recommended action

I don't think there's any reason for the `div` that wraps the `va-alert` to have `role="alert"`. I'd encourage you to remove that attribute.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by briandeconinck** [comment]
`launch-blocking` label removed -- [see discussion](https://github.com/department-of-veterans-affairs/va.gov-team/issues/98479#issuecomment-2557542702)

**Comment by kingbmike** [comment]
This is a form wide - platform code issue.
